src/basic/generate-af-list.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cpp -dM -include sys/socket.h - </dev/null | \
```

Comment:
This breaks cross-compiling:
`src/basic/af-from-name.gperf:25:18: error: 'AF_KCM' undeclared (first use in this function)`

The cross cpp must be used for this. The same is true for all other scripts that work like this.